src/routing/router.js

```diff
@@ -1,17 +1,24 @@
 import React from "react";
 import LoginForm from '../components/loginform/modules/index.js';
 import { BrowserRouter as Router, Routes, Route, Navigate } from 'react-router-dom';
+//FIXME: something tells me I shouldn't import sidebar like this!?
+import Sidebar from "../components/sidebar/Sidebar";
+import ProtectedRoutes from "./protectedRoutes";
 
 function CustomRouter() {
   return (
     <Router>
       <Routes>
+        <Route element={<ProtectedRoutes />}>
```

Comment:
It will be good to have sidebar on each protected route. I already put example in some comment so you can look at it. First protected route will be /dashboard and it will include sidebar.

Reply:
My suggestion is to create something like this 
```<AuthRoute path="/login" exact component={Login} />```

Reply:
and if you want protected route just type <ProtectedRoute path="/login" exact component={Login} /> 
and if you want something else you can create for example AdminRoute...